modules/jupyter-widget/src/playground/playground.js

```diff
@@ -38,6 +38,7 @@ export function initPlayground() {
         container,
         jsonInput: jsonProps,
         tooltip,
+        handleEvent: (name, payload) => sendEventViaTransport(transport, name, payload),
```

Comment:
Python would receive name and payload, name could be passed to a `switch`-like Python statement